Documentation/contributor-guide/release.md

```diff
@@ -100,3 +100,9 @@ etcd team
 7. Update the changelog to reflect the correct release date.
 8. Paste the release link to the issue raised at step 1 and close the issue.
 9. Crease a new stable branch through `git push origin release-${VERSION_MAJOR}.${VERSION_MINOR}` if this is a new major or minor stable release.
+
+#### Release known issues
+
+1. Network timeouts - It is rare, although possible for connection timeouts to occur when publishing etcd release images to `quay.io` or `gcr.io`. If this occurs, for `release-3.5` or newer it is known to be safe to rerun the release script command, and image uploads will gracefully resume.
```

Comment:
I believe it's the same for 3.4. But it'd be better to double check this.
```suggestion
1. Network timeouts - It is rare, although possible for connection timeouts to occur when publishing etcd release images to `quay.io` or `gcr.io`. If this occurs,  rerun the release script command, and image uploads will gracefully resume.
```

We also need to clearly mention,
- If the release script fails during pushing image, rerun the script with flag `--no-upload`;
- If the release script fails during uploading binaries, then remove all binaries from google buckets, and rerun the script.

Reply:
> I believe it's the same for 3.4. But it'd be better to double check this.

Read through the script for `release-3.4`, yes it's the same behaviour which is great, updated.

> If the release script fails during pushing image, rerun the script with flag --no-upload

Good catch yes we used that flag, updated.

> If the release script fails during uploading binaries, then remove all binaries from google buckets, and rerun the script.

Added section. No need to remove binaries. I confirmed we don't use object versioning on our bucket and `gsutil cp` will overwrite existing file (unless you use `-n` flag which we don't) so there is no chance incorrect files can remain.